komodo/lint.py

```diff
@@ -43,6 +45,20 @@ def _validate(pkg, ver, repo):
     return _kerr(pkg=pkg, version=ver, maintainer=repo[pkg][ver]['maintainer'])
 
 
+def lint_release_name(pkgfile):
+    relname = os.path.basename(pkgfile)
+    found = False
+    for py_suffix in "-py27", "-py36":
+        for rh_suffix in "", "-rhel6", "-rhel7":
+            if relname.endswith(py_suffix + rh_suffix + ".yml"):
```

Comment:
Should we not allow `relname.endswith(py_suffix + ".yml")`, or are we planning to switch everything over before merging this?

Reply:
`rh_suffix` can be, among other things, `""`.

Reply:
You are totally correct, I misread :disappointed: